src/main/java/org/prebid/server/bidder/improvedigital/proto/ImprovedigitalBidExtImprovedigital.java

```diff
@@ -0,0 +1,16 @@
+package org.prebid.server.bidder.improvedigital.proto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Value;
+
+@Builder(toBuilder = true)
```

Comment:
this class is used in unused file and also can be deleted

Reply:
This class can't be removed - it's needed by the code populating dealid from line buying type and line item id in ImprovedigitalBidder.java.

Reply:
Ok, but something should be done with this naming "ImprovedigitalBidExtImprovedigital"

Reply:
I thought hard about this but can't see what the new name could be. The name describes accurately the purpose: it's bid.ext.improvedigital object. Other bidders use the same convention (e.g. AppnexusBidExtAppnexus). I wouldn't want to introduce new convention just because `Improvedigital` is a little long.